src/scripts/components/elements/validation.js

```diff
@@ -66,4 +73,49 @@ export default class Validation {
       'h5p-reader-question-required-wrapper', 'h5p-reader-question-hidden'
     );
   }
+
+  /**
+   * Validate input/editor content
+   * @param {number} contentLength length of content
+   * @returns {boolean}.
+   */
+  validate(contentLength) {
+    const remainingChars = this.params.charactersLimit - contentLength;
+    this.params.fieldContainer.classList.remove('validation-error');
+    let isError = this.validateCharsLimit(contentLength) ? true : false;

```

Comment:
If the return value of `validateCharsLimit` is a `boolean` (it is), then using `let isError = this.validateCharsLimit(contentLength);` is sufficient. One does not need to explicitly return `true` if the return value is `true` and `false` if the return value is `false`.

If you think that you'd like this to be more "speaking", you can rename `validateCharsLimit` to something like `isCharLimitExceeded`. Then you'd have `let isError = this.isCharLimitExceeded(contentLength);` which should then be pretty self-explanatory and shorter.